python/ray/tune/syncer.py

```diff
@@ -303,32 +261,14 @@ def get_log_syncer(local_dir, remote_dir=None, sync_function=None):
     elif not remote_dir:
         sync_client = NOOP
     elif sync_function:
-        sync_client = _get_sync_client(sync_function)
+        sync_client = get_sync_client(sync_function)
     else:
-        sync_up = log_sync_template()
-        sync_down = log_sync_template(options="--remove-source-files")
```

Comment:
is the remove-source-files no longer used?

Reply:
oh ill revert this here it should go in the next PR, but the answer to your question is yes—I'll add an explanation for this in the other PR as well.